GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Remotes.cs

```diff
@@ -193,6 +182,46 @@ public void Reset()
                 }
             }
 
+            public bool FetchAndMerge()
+            {
+                if (Fetch())
+                {
+                    return Merge();
+                }
+
+                return false;
+            }
```

Comment:
Can this be simplified to `return Fetch() && Merge()`?

Reply:
Absolutely! Haha, I was actually planning on doing that, which is why I added bool return values everywhere, but forgot. Thanks :)

Reply:
Amended last commit with these changes.